packages/containers/src/SubHeaderBar/SubHeaderBar.container.js

```diff
@@ -35,68 +33,9 @@ class SubHeaderBar extends React.Component {
 
 	constructor(props) {
 		super(props);
-		this.onCancel = this.onCancel.bind(this);
-		this.onEdit = this.onEdit.bind(this);
-		this.onSubmit = this.onSubmit.bind(this);
-		this.onChange = this.onChange.bind(this);
 		this.onGoBack = this.onGoBack.bind(this);
 	}
 
-	onSubmit(event, data) {
-		this.props.setState(() => ({
-			editMode: false,
-		}));
-		if (this.props.onSubmit) {
-			this.props.onSubmit(event, data);
-		}
-		if (this.props.actionCreatorSubmit) {
-			this.props.dispatchActionCreator(this.props.actionCreatorSubmit, event, {
-				props: this.props,
-				data,
-			});
-		}
-	}
-
-	onCancel(event) {
-		this.props.setState(() => ({
-			editMode: false,
-		}));
-		if (this.props.onCancel) {
-			this.props.onCancel(event);
-		}
-		if (this.props.actionCreatorCancel) {
-			this.props.dispatchActionCreator(this.props.actionCreatorCancel, event, {
-				props: this.props,
-			});
-		}
-	}
-
-	onEdit(event) {
-		this.props.setState(() => ({
-			editMode: !this.props.state.get('editMode', false),
-		}));
-		if (this.props.onEdit) {
-			this.props.onEdit(event);
-		}
-		if (this.props.actionCreatorEdit) {
-			this.props.dispatchActionCreator(this.props.actionCreatorEdit, event, {
-				props: this.props,
-			});
-		}
-	}
-
-	onChange(event) {
```

Comment:
You removed all those methods but in the render you still refer to them instead of the props.